### PR TITLE
Wrong path of products thumbnails in back office

### LINKED ISSUE #14374: Image don't display in stock management & Category edition
**Describe the bug**
Image don't display in stock management 

**To Reproduce**
1. Go to Stocks in back office
2. See error

1. Go to Catalog -> category
2. Edit a category 
3. See the error

1. Go to Catalog -> Brand & Manufacturer
2. See the error

**Screenshots**
image

image

**Additionnal information**
PrestaShop version: Develop 
PHP version: N/A

### PR DESCRIPTION
**Pull request by matthieu-rolland**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | Fixes broken product image in backoffice > stock list page
| Type?         | bug fix
| Category?     | BO
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | Fixes #14374
| How to test?  | Go in backoffice > catalogue > stocks, and see that products thumbnails are displayed correctly, also in category edition page, and in brand and manufacturer list page

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/14543)

### CODE REVIEW

**Review by matks — DISMISSED** [review]

File: src/Core/Image/Parser/ImageTagSourceParser.php [hunk 1]
```diff
@@ -65,6 +65,6 @@ public function parse($imageTag)
             return null;
         }
 
-        return sprintf('%s%s', $this->shopRootUri, $path[1]);
+        return sprintf('/%s', $path[1]);
```
Comment (matks) [inline src/Core/Image/Parser/ImageTagSourceParser.php:68]: Can we check that this does not break the image display of Languages listing `/admin-dev/index.php/improve/international/languages/` ? 
I dont want to introduce a breaking change for PR https://github.com/PrestaShop/PrestaShop/pull/10864

Comment (PierreRambaud) [reply src/Core/Image/Parser/ImageTagSourceParser.php:68]: watch out if you're not running under a virtualhost!

Comment (matthieu-rolland) [reply src/Core/Image/Parser/ImageTagSourceParser.php:68]: I have a virtualhost and it doesn't break the image display of Languages listing (if you're talking about the little flag in the language list).

From the behavior I've seen on my local environment:

Both in the vuejs and in the ImageTagSourceParser.php file, the issue was that the shop root uri/ base url was not needed in the image paths. Did I take the problem backward , and the issue is something else?

Here is a correct image path on my local environment:

 http://myprestashop.localhost/img/tmp/product_mini_2.jpg

@matks @PierreRambaud

Comment (matthieu-rolland) [reply src/Core/Image/Parser/ImageTagSourceParser.php:68]: @matks @PierreRambaud 

Ok I created a new prestashop environment without a proper virtualhost, and my fix still works, even in the language listing page.

This is the kind of img path I have in this case:

http://localhost/test/PrestaShop/img/tmp/product_mini_1.jpg

Do you have ideas of other things I should watch for?

**Comment by matthieu-rolland** [comment]
@PrestaShop/prestashop-core-developers 
This PR is still waiting for approval :)

**Review by PierreRambaud — APPROVED** [review]

**Pull request merged**